### ISSUE TITLE
Search for NPCs in list by name

### ISSUE THREAD

**Issue by Leo40Git** [taedixon/boosters-lab#17]
Suggestion by »_Onward#3840 on Discord.
Can probably get away with simply iterating over every NPC and using String.contains on short and long names.

**Comment by taedixon** [comment]
I assume you're suggesting something like a "filter list" by this?

**Comment by Leo40Git** [comment]
Yeah, something like that. I don't know where'd you'd put the text field to enter the filter, though.
